### DISCUSSION TITLE
Defining sites excluded from authentication

### DISCUSSION THREAD

**Discussion by wsw70** [authelia/authelia#6323] [Q&A] [answered]
The [documentation ](https://authelia.com/configuration/security/access-control/#default_policy) states 

> Sites which you do not wish to secure at all with Authelia should not be configured in your reverse proxy to perform authentication with Authelia at all for performance reasons

I self-host several services, not particularly performance intensive so **I would like to know whether having the exclusions in Authelis is a really bad idea, or just a not-too-bright one** (and how performance can be hindered).

The reason for the change is that I would like to have the exclusions close to Authelia so that the exclusions are contextualized and all in one place. Currently I have separate Caddy configuration files for each of my containers and a common block for Authelia which is applied in these separate Caddy files. This has several drawbacks (I can forget to protect, I need to search though the files to check what is protected etc.)

The documentation uses `bypass` extensively in the examples, so I have hope that the warning is for some really heavy-duty sites that get a lot of authentication requests.

**Comment by james-d-elliott** [comment, marked as answer]
You can do either option technically however the most efficient is excluding it
